src/Builder/ConditionalLookup.php

```diff
@@ -108,7 +109,15 @@ class: new Node\Name\FullyQualified('Kiboko\\Component\\Bucket\\AcceptanceResult
                     ),
                 ),
                 new Node\Stmt\If_(
-                    cond: $parser->parse('<?php ' . $this->interpreter->compile($condition, ['input', 'lookup', 'output']) . ';')[0]->expr,
+                    cond: (function () use ($parser, $condition) {
+                        if (is_string($condition)) {
+                            return new Node\Scalar\String_($condition);
+                        }
+                        if ($condition instanceof Expression) {
+                            return $parser->parse('<?php ' . $this->interpreter->compile($condition, ['input']) . ';')[0]->expr;
```

Comment:
ca peut etre remplacé par `return new Node\Scalar\String_($this->code),` comme dans `Builder/Capacity/Extractor/All.php` ligne 85 ?